publishing/ui-metadata/README.md

```diff
@@ -0,0 +1,51 @@
+## UI metadata
+
+By convention, a blendable web app must provide a metadata JSON that responds to the `app-name` blending token.
+
+System apps and navigation apps like [Launchpad](https://github.com/StarcounterApps/Launchpad) will display a link to the entry point URI using the app name from this metadata. They will also use the icon, if provided.
+
+The response JSON can consist of the following properties:
+
+|Property|Explanation|
+|---|---|
+|`name`| **Required.** Human readable app name. |
+|`description`| Optional. Short (single sentence) description of the app. |
+|`html`| Optional. URI to a view that contains the app icon. |
+
+Sample `MainHandlers.cs` ([source](https://github.com/StarcounterApps/CssVariablesManager/blob/develop/src/CssVariablesManager/Api/MainHandlers.cs#L13)):
```

Comment:
I have changed CSSVariableManager as an example in the "Entry point URI" page.

Here, I don't have a better example though, because other apps don't have SVG icons yet.